lib/pysquared/Field.py

```diff
@@ -4,42 +4,38 @@
 Authors: Nicole Maggard, Michael Pham, and Rachel Sarmiento
 """
 
-import traceback
```

Comment:
Noting for the future that we talked about removing traceback and we couldn't find a reason to keep it. We tried both in repl and found that reporting back the exception var `e` gave the same output but maybe we missed something. 🤷